app/__init__.py

```diff
@@ -0,0 +1,118 @@
+from flask import Flask, request
+from flask_restx import Resource, Api, fields, marshal
+import werkzeug.exceptions
+from db.TinyDBManager import TinyDBManager, TinyDBDocumentNotFoundException
+
+
+def create_app(db=TinyDBManager()):
```

Comment:
Why is this a parameter?
```suggestion
def create_app():
    db = TinyDBManager())
```

Reply:
I've kept it as a parameter for when we write tests for this API. By keeping it as a param, I can pass in a Mock object.